.github/workflows/main_ci.yml

```diff
@@ -101,23 +125,36 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [windows-latest, ubuntu-latest, macos-latest, macos-11]
```

Comment:
I would note that there's a slightly subtle difference between this setup and the separate build below.  In the current setup, the build platform is `macos-latest`, but with `MACOSX_DEPLOYMENT_TARGET: 10.11`.  Here we build directly on 10.11.  I suspect that the. current setup is more reflective of how people actually build things with compatibility for old platforms (as opposed to keeping around machines actually running the old OS).  But I think this is probably OK in terms of catching incompatibilities / failures.

Reply:
I see now. I might have fixed that.

Reply:
I did fix this, and I was able to open the door to building several macOS targets.